shared/test_plot_band_intensity.py

```diff
@@ -0,0 +1,167 @@
+# This code is used to plot the standarized band intensity for each of the classes
+# x-mean(x)/sd(x)
+# Usage: python3 test_get_clean_debris.py -f [shp_filename with debris information]
+# python3 test_plot_band_intensity.py -f ../data/vector_data/2005/nepal/data/Glacier_2005.shp
+# Output:   The output is saved as test.png in the current directory
+import sys
+sys.path.append('../')
+import rasterio
+import geopandas
+import argparse
+import pickle
+import glob
+import numpy as np
+import rasterio
+
+import torchvision.transforms as T
+from src.utils import get_debris_glaciers
+from src.utils import get_mask
+from src.utils import slice_image
+from src.preprocess import save_slice
+import matplotlib 
+import matplotlib.pyplot as plt
+from rasterio.mask import mask as rasterio_mask
+from sklearn.preprocessing import Normalizer
+
+np.random.seed(7)
+
+if __name__ == "__main__":
+    n_points = 5000
+
+    norm_data_file = "../data/normalization_data.pkl"
+    norm_data = pickle.load(open(norm_data_file, "rb"))
+    mean, std = norm_data["mean"], norm_data["std"]
+    norm_mean, norm_std = mean.numpy(), std.numpy()
+    norm_mean = np.append(norm_mean[0:5],norm_mean[5])
```

Comment:
Why do you have to do this append? Why not just subset :6?

Reply:
The landsat 7 band with index 5 has a wavelength of 10.40-12.50 μm while bands with index 0-4 and 6 all fall within wavelength of 0.45-2.35. I am omiting the higher wavelength from the graph so it is more readable. An alternative to this would be to add a break in between 2.35 and 10.40 in the x axis and plot all 7 points.

Reply:
Ah okay, this makes sense, thank you.